grails-app/domain/org/pih/warehouse/order/OrderItem.groovy

```diff
@@ -124,6 +124,10 @@ class OrderItem implements Serializable, Comparable<OrderItem> {
         }
     }
 
+    def hasShipmentAssociated() {
+        return shipmentItems?shipmentItems.size()>0:false
```

Comment:
I think you forgot about ?. It's also better to add spaces to it's more clear.

Reply:
There's no need for ?. since it is if statement already, but I'll add the spaces for better readability.